app/blobs/new-shape.jsx

```diff
@@ -22,12 +22,6 @@ export function NewShape(props) {
         setLastMutationTime(Date.now());
     };
 
-    useEffect(() => {
-        if (!blobData) {
-            randomizeBlob();
-        }
-    }, [blobData]);
-
```

Comment:
this was an existing unnecessary and non-idiomatic use of `useEffect`... and for some reason stopped working anyway, so I refactored it away